cmd/playground/window.go

```diff
@@ -0,0 +1,72 @@
+package playground
+
+import (
+	"time"
+
+	"github.com/odpf/optimus/job"
+)
+
+type state struct {
```

Comment:
after checking, some of the responsibility of this `state` is handled directly by the `model` struct. if we want to retain this in a separate struct, then let this struct has its own responsiblity. like changing the value of `window1/2` or something instead of allowing `model` to do that. however, if `model` is still to handle this, then better merge this `state` struct to that.